packages/docClient/src/Client.ts

```diff
@@ -38,6 +52,9 @@ export class Client extends EventEmitter<ServiceEvent, PossibleServiceEvent['dat
 
     async connect() {
         this.connection = new WsConnection({ url: this.config.url, token: this.config.token });
+        this.connection.on('connect', this.handleConnectionEvent({ event: 'connect', data: null }));
+        this.connection.on('disconnect', this.handleConnectionEvent({ event: 'disconnect', data: null }));
```

Comment:
I don’t particularly like this hack, maybe you have any ideas on how to get rid of it?

It does work though
<img width="806" alt="Screenshot 2023-12-13 at 01 49 02" src="https://github.com/a-la-fois/a-la-fois/assets/43999040/156e9442-c225-4f27-858b-7d029341de12">

Reply:
Nah, I think this is fine